articat/utils/utils.py

```diff
@@ -89,7 +89,9 @@ def download_artifact(artifact: FSArtifact, local_dir: PathType) -> str:
     prefix = artifact.main_dir
     # Note: glob results don't have fs scheme
     prefix_no_scheme = re.sub(FSArtifact._fs_scheme_regex, "", prefix)
-    to_copy = src_fs.glob(artifact.files_pattern)
+    # Root directory can be included in glob results if a trailing ** is used (which it often is)
```

Comment:
Nice, I remember this being inconsistent across FSs, and I remember disliking this logic a lot, is there any chance to simplify the current implementation now given the (assumption!) more consistent glob results?

Reply:
Mm, I gave it another read-through, and I don't see too much wrong with it. If anything, I guess I could see using the more consistent glob behavior to change what glob patterns were used upstream, but that sounds like potentially a complicated effort to track down callsites.. and moreover, I don't think it would rank high on the list of company objectives 😉